File: nav1/src/main/java/com/example/nav1/MainActivity.java
package com.example.nav1;

import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.NavController;
import androidx.navigation.fragment.NavHostFragment;
import androidx.navigation.ui.NavigationUI;

import android.os.Bundle;

import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        NavHostFragment navHostFragment= (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.hostFragment);
        NavController navController=navHostFragment.getNavController();
        BottomNavigationView bottom=findViewById(R.id.bottom);
        NavigationUI.setupWithNavController(bottom,navController);
    }
}

File: share/src/main/java/com/example/share/MainActivity.kt
package com.example.share

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.fragment.NavHostFragment

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val navHostFRagment=supportFragmentManager.findFragmentById(R.id.menu1_fragment) as NavHostFragment
        val nav=navHostFRagment.navController


    }
}
